### ISSUE TITLE
Race condition if DBus process unexpectedly goes away

### ISSUE THREAD

**Issue by tmontes** [nununo/pyVela2017#34]
Following up on #30, #31, #32, and playing around with starting, stopping, process relationships and their unexpected exits, found yet another "ugly way" things can fail (as in, it will hang in there for ever).

Reproducing:
* Processes running: `dbus-run-session`, `dbus-daemon`, `python` and 4x `omxplayer.bin`.
* `dbus-daemon` suddenly goes away.
* Result:
  * No further `candle2017.py` to `omxplayer.bin` communication is possible:
  * Interaction is gone: no more video play/pause/fade control.
  * Stops may fail waiting forever on the `omxplayer.bin` processes to go away from DBus.
  * `omxplayer.bin` unexpected stops aren't properly handled: player manager respawns then, but waits forever for them to appear on DBus, thus never pauses them, etc.)

Without detecting DBus disconnections, the race condition solution is to implement a timeout on the last step above. Stop will work, albeit somewhat slowly, depending on the timeouts.

Detecting DBus disconnections, there are two possible solutions:
* DBus manager "marks" all names as gone, including ones that it will be asked about in the future; this fixes the race condition and ensures a fast stop.
* Player manager initiates an "emergency" top, avoiding `_stop_via_dbus()` and going directly to `_stop_via_sigterm()`; this avoids the race condition and ensures a fast stop.

DBus disconnections can be detected with:
```
from txdbus import client

dbus_conn = yield client.connect(...)
dbus_conn.notifyOnDisconnect(callable_receiving_two_args)
```

**Comment by nununo** [comment]
Closed by #35

**Comment by tmontes** [comment]
> Closed by #35

...but GitHub did not close it automatically. Let's do it by hand as "God Intended". :)
